### ISSUE TITLE
[Feature]: Can I use this project to open a GUI broswer(headless=false)?

### ISSUE THREAD

**Issue by wade-liwei** [playwright-community/playwright-go#516]
Can I use this project to open a GUI broswer(headless=false)?
any browser is ok.

right now I based on  xfce4  novnc  can not open Chromium.

**Comment by mxschmitt** [comment]
Yes, headed mode (`headless=false`) is fully supported on Ubuntu — playwright-go just wraps the Playwright driver. A few pointers:

- **Launch headed:**
  ```go
  browser, err := pw.Chromium.Launch(playwright.BrowserTypeLaunchOptions{
      Headless: playwright.Bool(false),
  })
  ```
- **Ensure a valid `DISPLAY`** is exported in the same process that runs your Go binary (e.g. `export DISPLAY=:0` for your VNC X server).
- **Easiest on a headless server:** run under a virtual framebuffer — `xvfb-run -a ./your-program`.
- **Install OS deps** if the browser won't start: `playwright install-deps` (missing libs like `libgbm.so.1` are the usual cause).
- If it still fails, capture Chromium's stderr — it normally names the missing `.so` or a `$DISPLAY` problem.

Closing as this is an environment/usage question rather than a library bug — feel free to reopen with the exact error output if the above doesn't help.
